### DISCUSSION TITLE
主机记录值能改么?

### DISCUSSION THREAD

**Discussion by Vejun** [sanjusss/aliyun-ddns#53] [Q&A]
大大,麻烦你一下,就是这个能更改主机记录值,它默认是设置的@,但是用@的话经常无法使用域名登录,如果设置成WWW的话就能打开,我想最保险的就是设置2个一个@一个www 可以这样操作么?有 详细步骤么?救救孩子

**Comment by sanjusss** [comment]
https://github.com/sanjusss/aliyun-ddns#docker
![image](https://user-images.githubusercontent.com/8220722/140687746-7f8471e4-3f8d-4ab7-a6c9-99734c39da47.png)
只想设置多个域名，可以设置为
```
a.com,www.a.com
```

**Reply by Vejun** [reply]
感谢大大!   把这给忘了 ......
